components/transfer/index.en-US.md

```diff
@@ -25,7 +25,7 @@ To transfer the elements between two columns in an intuitive and efficient way.
 | onSelectChange | A callback function which is executed when selected items are changed. | Function(sourceSelectedKeys, targetSelectedKeys) | |
 | listStyle | A custom CSS style used for rendering the transfer columns. | Object |  |
 | className | A custom CSS class. | String |  |
-| titles | A set of titles that are sorted from left to right. | Array | ['源列表', '目的列表'] |
+| titles | A set of titles that are sorted from left to right. | Array | ['Source list', 'Target list'] |
 | operations | A set of operations that are sorted form top to bottom. | Array | [] |
```

Comment:
` | operations | A set of operations that are sorted from top to bottom. | Array | [] |`